integration_test/testdata/foo.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: test-foo
+spec:
+  version: "v0.1.0"
+  shortDescription: A valid plugin for integration tests
+  platforms:
+    - uri: https://foo.bar/foo.tar.gz
+      sha256: e72bad12ce8faa84aae929894acad9d759a40d63befde2a9dcf6ba12487cc362
+      bin: foo
+      files:
+        - from: ./foo
+          to: "."
+      selector:
+        matchExpressions:
+        - key: os
+          operator: In
+          values: ["darwin", "linux"]
+    - uri: https://foo.bar/foo.tar.gz
```

Comment:
I recommend removing the windows section. (We strive to keep testdata manifests to only what's being tested.)

You can make a select-all OS selector like:

```
matchLabels:
  os: {}
```

this should match all OSes.

Reply:
Yes, you are right. My thinking was to make sure that the test would run on Windows (also see #131), but that does not depend on the file ending.

Reply:
I think regardless we make a symlink called kubectl-foo.exe anyway. That said I'm not sure if that would actually work when the target file is not named .exe/.com/.bat/.ps1 etc. If you create a `foo.tar.gz` with foo.sh and foo.bat, this would probably be more straightforward.

Reply:
Ok, so I added a windows version of the test-foo plugin. I think that should do it.